### ISSUE TITLE
Shouldn't this be a web application with a database instead of a plain csv file.

### ISSUE THREAD

**Issue by the-kaustubh** [mukta-strot/marathi-shabd#4]
A web app so that layman users can search for words that they need, or adding new words could be simplified.

Possible Problem:
Everytime a new english word has to be added we have to make changes to csv file and open a PR. Even if in future a github.io page is added, it will still be diffiicult to work with a single csv file.

Proposed Solution:
Web app with its database which can be used to fetch words through a Github page (frontend). And this app can also provide a simple UI to add new English words and their alternative Marathi words.

**Comment by sanketgarade** [comment]
yeah even i thought of this, but i thought of making a simpler solution first mainly for below reasons - 

- I am not a web developer or even a application programmer so I have no experience in these and the languages/technologies they need. My exp is in firmware (embedded C) and hardware.
- due to work and personal commitments, at the moment, I may not be able to much devote time to learning new stuff.
- i know markdown and some basic python, and have exp in making a basic static github pages site. The simple solution that I have in mind can be done using this with minimal effort in learning new stuff.

Having said that, I am ok with someone developing a better solution if they can. Only requirement is that it should be lightweight, static (as much as possible) and bloat free. take a look at [this recipe site](https://based.cooking/) and [this lyrics site](https://lyrics.github.io/) for reference.

**Comment by the-kaustubh** [comment]
I understand, one might not be able to give enough time and has his/her own work to do. But I think that is where open-source could help. Many people who care about the language and/or interested in contributing will contribute to the project.

Also, the reason for opening this particular issue was not only to suggest but as I am myself a web developer I would love to contribute to this project. Also in my opinion the project should be catered specifically for the users to actually use Marathi words instead of English ones.

To sum up, I have started working on a simple bloat-free website (although it is not static) and might open a PR soon. I would be happy to know your response.

**Comment by sanketgarade** [comment]
Sure. Go ahead. I hope you can add comments in the code so that I can understand it, since it may be new to me. Also which language will you be using?

**Comment by the-kaustubh** [comment]
I have already opened a PR. I will be making a static website for now using HTML CSS and Javascript

**Comment by sanketgarade** [comment]
Ok. Let me know once the basic site is ready to be hosted. So that we can then do the GitHub pages settings and try it out. Once all ok then we can merge into the main branch.

**Comment by sanketgarade** [comment]
I have a question. github pages [documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) says it can host only static website. So will your proposed solution in the top comment (github pages frontend with JS and some database at backend) work?
